operator/validator/controller.go

```diff
@@ -151,6 +157,8 @@ type controller struct {
 	// nonCommittees is a cache of initialized nonCommitteeValidator instances
 	nonCommitteeValidators *ttlcache.Cache[spectypes.MessageID, *nonCommitteeValidator]
 	nonCommitteeMutex      sync.Mutex
+
+	indicesChange chan bool
```

Comment:
this could be an empty struct `indicesChange chan struct{}` since we never use the boolean?

Reply:
see https://github.com/bloxapp/ssv/pull/1044#discussion_r1273232868